srcs/parsing/lexer.c

```diff
@@ -19,12 +19,12 @@ t_error	lexer(char *line)
 	t_error		exit_code;
 
 	minishell = get_minishell(NULL);
-	if (minishell->nb_cmds)
-	{
+	if (minishell->instructions)
 		free_split(minishell->instructions);
+	if (minishell->child_pids)

```

Comment:
wait what? Shouldn't it be if minishell->cmd_table?

Reply:
yup